spec/requests/v0/profile/addresses_spec.rb

```diff
@@ -348,322 +332,296 @@
 
-          expect(response).to have_http_status(:forbidden)
+            expect(response).to have_http_status(:forbidden)
+          end
         end
       end
-    end
 
-    context 'with a validation issue' do
-      before do
-        address.address_pou = ''
-      end
+      context 'with a validation issue' do
+        before do
+          address.address_pou = ''
+        end
 
-      it 'creates a PII log for the validation error' do
-        post('/v0/profile/addresses', params: address.to_json, headers:)
+        it 'creates a PII log for the validation error' do
+          post('/v0/profile/addresses', params: address.to_json, headers:)
 
-        log = PersonalInformationLog.last
-        log_data = log.data
-        expect(log_data['address_line1']).to eq(address.address_line1)
-        expect(log_data['address_pou']).to eq(address.address_pou)
-        expect(log.error_class).to eq('VAProfile::Models::V3::Address ValidationError')
-      end
+          log = PersonalInformationLog.last
+          log_data = log.data
+          expect(log_data['address_line1']).to eq(address.address_line1)
+          expect(log_data['address_pou']).to eq(address.address_pou)
+          expect(log.error_class).to eq('VAProfile::Models::V3::Address ValidationError')
+        end
 
-      it 'matches the errors schema', :aggregate_failures do
-        post('/v0/profile/addresses', params: address.to_json, headers:)
+        it 'matches the errors schema', :aggregate_failures do
+          post('/v0/profile/addresses', params: address.to_json, headers:)
 
-        expect(response).to have_http_status(:unprocessable_entity)
-        expect(response).to match_response_schema('errors')
-        expect(errors_for(response)).to include "address-pou - can't be blank"
-      end
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(response).to match_response_schema('errors')
+          expect(errors_for(response)).to include "address-pou - can't be blank"
+        end
 
-      it 'matches the errors camel-inflected schema', :aggregate_failures do
-        post('/v0/profile/addresses', params: address.to_json, headers: headers_with_camel)
+        it 'matches the errors camel-inflected schema', :aggregate_failures do
+          post('/v0/profile/addresses', params: address.to_json, headers: headers_with_camel)
 
-        expect(response).to have_http_status(:unprocessable_entity)
-        expect(response).to match_camelized_response_schema('errors')
-        expect(errors_for(response)).to include "address-pou - can't be blank"
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(response).to match_camelized_response_schema('errors')
+          expect(errors_for(response)).to include "address-pou - can't be blank"
+        end
       end
     end
-  end
-
-  describe 'PUT /v0/profile/addresses v2' do
-    before do
-      Flipper.enable(:va_v3_contact_information_service)
```

Comment:
Same question here: you don't need `allow(Flipper).to receive(:enabled?)...` for these tests?

Reply:
The flipper is enabled on [line 337](https://github.com/department-of-veterans-affairs/vets-api/blob/b4ed91d588872d777815af3919818d2668e4515b/spec/requests/v0/profile/addresses_spec.rb#L337)

Reply:
ohh, okay. I see now. It was hard to discern the indentation.